source/docs/advanced/payments/adyen.md

```diff
@@ -189,6 +210,79 @@ Add the following line to your `web/theme/modules/_modules.scss` file to load th
+
+<blockquote class="note">
+Please keep in mind that Adyen payment methods depends on the Customer country, the Cart amount and the Store currency. Different contexts could display different payment methods in the checkout.
+</blockquote>
+
+## Magento2 module
+
+<blockquote class="feature--new">
+  _Since version 2.3.0 (developer preview since 2.1.0)_
+</blockquote>
+
+<blockquote class="note">
+**Note** This integration is aimed at being transparent for administrators and developers. That is why we haven't duplicated documentation from existing Magento resources. Please <span class="intercom-launcher">[contact us](mailto:support@front-commerce.com)</span> if you need further assistance.
+</blockquote>
+
+Front-Commerce is compatible with [Adyen's official Magento headless integration](https://docs.adyen.com/plugins/magento-2/magento-headless-integration) in its native version.
+
+This integration is slightly different from [traditional Magento2 headless payments](/docs/magento2/headless-payments.html) in that sense that it contains a "companion component" in Front-Commerce. The component allows to display and Authorize payments from the checkout page, using [Adyen's Web Drop-in integration](https://docs.adyen.com/online-payments/drop-in-web) on the front-end. No redirection to other payment platform is involved unless its absolutely necessary, the Customer remains on the Front-Commerce store.
+
+Here is how to set this payment method up.
+
+### Install and configure the `Adyen_Payment` Magento2 extension (7.2+)
+
+Follow [Adyen's documentation](https://docs.adyen.com/plugins/magento-2) to install and configure the official Magento2 extension for Adyen payments. The minimum supported version is [7.2.0](https://github.com/Adyen/adyen-magento2/releases/tag/7.2.0).
+
+You must configure the "Payment Origin URL" in the "Advanced: PWA" with your Front-Commerce URL **for each of your stores**.
+
+<blockquote class="important">
+**IMPORTANT** please note that enabling or disabling payments in Magento's admin area has no effect on payment methods visible on the storefront. This is a *feature* from the module: every active payment methods in Adyen will be available. **You either have to filter them in the frontend or from your Adyen account.**
+</blockquote>
+
+### Add the Adyen client key in the environment
+
+You need to define the `FRONT_COMMERCE_ADYEN_CLIENT_KEY` environment variable so that it contains [your Adyen client key](https://docs.adyen.com/development-resources/client-side-authentication) for the domains of your Front-Commerce stores:
+
+```sh
+# In .env
+FRONT_COMMERCE_ADYEN_CLIENT_KEY=live_32charactersstring
+# the Adyen client key is prefixed with live_ or test_
+```
+
+### Register the Adyen payment module in Front-Commerce
+
+[Same as above](#Register-the-Adyen-payment-module-in-Front-Commerce)
```

Comment:
this step is actually different than the previous one since it's not the same module and is only for Magento2